### DISCUSSION TITLE
Discrepancy in the infiltration values between PySWMM and PCSWMM

### DISCUSSION THREAD

**Discussion by SaraBa87** [pyswmm/pyswmm#478] [Q&A] [answered]
I ran an *.inp file in both PCSWMM and PySWMM, and I noticed a discrepancy in the infiltration values between the two software packages. To retrieve statistical data for subcatchment 10, I used the following code:

with Simulation('Test.inp') as sim:
    for step in sim:
        pass
    print(Subcatchments(sim)["S10"].statistics)

The results are as follows:
{'evaporation': 0.0, 'infiltration': 10113.609875706654, 'peak_runoff_rate': 1.5025912004190747, 'precipitation': 95.99999999999997, 'runoff': 9110.16575709845, 'runon': 0.0}

However, the amount of infiltration obtained from PCSWMM for this subcatchment is 49.93 mm. I'm curious to understand why there is a difference between these two infiltration values.

Thank you,
Sara

**Comment by bemcdonnell** [comment]
@SaraBa87 its probably a units thing.  If I remember correctly the stats are all in ft3. Can you tell me the surface area of the subcatchment ?

**Reply by SaraBa87** [reply]
The units in my *.inp file is SI. As far as I understood the stats are in SI, as well. The amount of runoff and pick runoff rate obtained from PCSWMM are as follows which are consistence with PySWMM:
Runoff volume (ML) = 9.11
Peak runoff (m3/s) = 1.5

The area of this subcatchment is 20.254 ha

**Comment by karosc** [comment, marked as answer]
@SaraBa87, the infiltration units are indeed in m³. If I'm doing my math right, 10113.6098757066 m³ spread over 202,540 m² is 49.9934mm, which matches the PCSWMM result.

**Reply by bemcdonnell** [reply]
One point @karosc, minus 1 point @bemcdonnell. I knew it was cubic volume - i forgot about the US/SI 😇😇😇

**Reply by SaraBa87** [reply]
Thank you!
